tcms/bugs/tests/test_permissions.py

```diff
@@ -58,6 +59,32 @@ def verify_post_with_permission(self):
         self.assertContains(response, 'BUG-%d' % last_bug.pk)
 
 
+class TestEdit(tests.PermissionsTestCase):
+    permission_label = 'bugs.change_bug'
+    http_method_names = ['get', 'post']
+
+    @classmethod
+    def setUpTestData(cls):
+        bug = BugFactory()
+
+        cls.url = reverse('bugs-edit', args=(bug.pk,))
+        cls.post_data = {
+            'summary': 'An edited summary'
+        }
+
+        super().setUpTestData()
+
+    def verify_get_with_permission(self):
+        response = self.client.get(self.url)
+
+        self.assertContains(response, _('Edit bug'))
+        self.assertContains(response, 'bugs/js/mutable.js')
```

Comment:
Instead of checking the existence of JavaScript file this scenario can do better and verify that the page is indeed the edit page for self.bug. The original summary will be shown on the page.